src/cursor/cur_backup_incr.c

```diff
@@ -52,6 +52,7 @@ __curbackup_incr_blkmod(WT_SESSION_IMPL *session, WT_BTREE *btree, WT_CURSOR_BAC
     if (v.len)
         F_SET(cb, WT_CURBACKUP_HAS_CB_INFO);
     __wt_config_subinit(session, &blkconf, &v);
+    __wt_verbose(session, WT_VERB_BACKUP, "__wt_config_next \"%s\"", config);
```

Comment:
I don't think we want to keep this. If you do it needs to be more informative.

Reply:
removed